pkg/cli/admin/release/extract_tools.go

```diff
@@ -307,6 +309,12 @@ func (o *ExtractOptions) extractCommand(command string) error {
 	refExact.Ref.ID = release.Digest.String()
 	exactReleaseImage := refExact.String()
 
+	// will extract in parallel
+	opts := extract.NewOptions(genericclioptions.IOStreams{Out: o.Out, ErrOut: o.ErrOut})
+	opts.ParallelOptions = o.ParallelOptions
+	opts.SecurityOptions = o.SecurityOptions
+	opts.OnlyFiles = true
+
```

Comment:
the opts are moved from below, to use to verify user-given release:tag-tool image exists.